src/main/java/teammates/ui/webapi/action/GetStudentProfilePictureAction.java

```diff
@@ -1,25 +1,75 @@
 package teammates.ui.webapi.action;
 
+import org.apache.http.HttpStatus;
+
+import teammates.common.datatransfer.attributes.StudentAttributes;
+import teammates.common.datatransfer.attributes.StudentProfileAttributes;
+import teammates.common.exception.UnauthorizedAccessException;
 import teammates.common.util.Const;
+import teammates.common.util.StringHelper;
 
 /**
  * Action: serves a profile picture that is stored in Google Cloud Storage.
  */
 public class GetStudentProfilePictureAction extends Action {
 
+    /** String indicating ACCESS is not given. */
+    public static final String UNAUTHORIZED_ACCESS = "You are not allowed to view this resource!";
+
+    /** String indicating student not found. */
+    public static final String STUDENT_NOT_FOUND = "No student found for given parameters";
+
+    /** String indicating profile picture not found. */
```

Comment:
Mentioning the type `String` in the comment is redundant?

Reply:
Would `Message` be better?

Something like `/** Message indicating student not found. */`

Reply:
`Indicates the student is not found`?

Reply:
Yup that is better! I have updated accordingly.